packages/harvester/harvester-cloud-provider/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/harvester/charts/releases/download/harvester-cloud-provider-0.1.13/harvester-cloud-provider-0.1.13.tgz
-version: 100.0.3
+url: https://github.com/harvester/charts/releases/download/harvester-cloud-provider-0.1.14/harvester-cloud-provider-0.1.14.tgz
+version: 101.0.3
```

Comment:
Charts for Rancher v2.6 should be in 100.x.y range, so we should probably bump it to `100.0.4` instead.